harvester/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM viderum/ckan-ng-harvest:latest
```

Comment:
Why do we need this Dockerfile?

Reply:
My first try to get the CKAN API KEY was via Dockerfile. I'll delete this file